docs/compliance/AI_GOVERNANCE_RMF.md

```
@@ -0,0 +1,67 @@
+# FFE Agentic AI Governance (NIST AI RMF)
+
+## Purpose
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)